src/app/system/system-routing.module.ts

```diff
@@ -34,27 +38,49 @@ const routes: Routes = [
       path: 'system',
       data: { title: extract('System'), breadcrumb: 'System' },
       children: [
-      {
```

Comment:
Remove unnecessary changes.

Reply:
There are no unnecessary changes in this file as of now, multiple lines are being removed because it was the first component.

Reply:
There are changes to indentation. Refactoring the code is not a part of this issue.

Reply:
Sorry, but I do not see any indentation related changes.

Reply:
There are some changes to spacing.